src/test/java/org/ambraproject/wombat/service/PeerReviewServiceImplTest.java

```diff
@@ -36,156 +35,6 @@
 
   private PeerReviewServiceImpl service = new PeerReviewServiceImpl();
 
-  @Test
```

Comment:
Consolidated testing of transformed content in `testTransformXmlToHtml` and removed all of this mockery.